content-scripts/prototype-handler.js

```diff
@@ -21,6 +21,40 @@ function injectPrototype() {
       return oldBind.apply(this, args);
     }
   };
+
```

Comment:
this would only run on project pages?

Reply:
Oof, true...

Reply:
Hadn't hought of that. Just a sec.

Reply:
How do you suggest I move this? Should I make a new `__scratchAddons` object in the window?

Reply:
move it to `cs.js`

Reply:
Okay.

Reply:
I am thinking about adding a `window.__scratchAddonsEventListeners` variable, sound okay?

Reply:
no, use scratchAddons.localState.eventListeners

Reply:
Okay.

Reply:
Umm.... No can do? It is an object in the bg, this needs to be for content-scripts.

Reply:
oh wait

move it to `content-scripts/inject/module.js` and use `scratchAddons.eventListeners`